Chinese_Remainder_Theorem/Readme.md

```diff
@@ -0,0 +1,108 @@
+# Chinese Remainder Theorem
+
+If m<sub>1</sub>, m<sub>2</sub>, .., m<sub>k</sub> are pairwise relatively prime positive integers, and if a<sub>1</sub>, a<sub>2</sub>, .., a<sub>k</sub> are any integers, then the simultaneous congruences x ≡ a<sub>1</sub> (mod m<sub>1</sub>), x ≡ a<sub>2</sub> (mod m<sub>2</sub>), ..., x ≡ a<sub>k</sub> (mod m<sub>k</sub>) have a solution, and the solution is unique modulo m, where m = m<sub>1</sub>m<sub>2</sub>⋅⋅⋅m<sub>k</sub> .
```

Comment:
Please add references for all this information at the bottom of Read me. We cannot use the examples directly without quoting any reference.